tools/depends/target/crossguid/Makefile

```diff
@@ -30,14 +32,14 @@ endif
 
 LIBDYLIB=$(PLATFORM)/lib$(LIBNAME).a
 
+include ../../download-files.include
```

Comment:
This seems viable to move into the include CROSSGUID-VERSION line.

Did some testing, and HASH_TOOL and the flags set just above still pass through to the download-files $(TARBALLS_LOCATION)/$(ARCHIVE).$(HASH_TYPE) make target correctly